main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/go-gitea/gitea/modules/markup"
+	"github.com/gophish/gophish/config"
+	"golang.org/x/crypto/md4"
+)
+
+var validConfig = []byte(`{
+	"admin_server": {
+		"listen_url": "127.0.0.1:3333",
+		"use_tls": true,
+		"cert_path": "gophish_admin.crt",
+		"key_path": "gophish_admin.key"
+	},
+	"phish_server": {
+		"listen_url": "0.0.0.0:8080",
+		"use_tls": false,
+		"cert_path": "example.crt",
+		"key_path": "example.key"
+	},
+	"db_name": "sqlite3",
+	"db_path": "gophish.db",
+	"migrations_prefix": "db/db_",
+	"contact_address": ""
+}`)
+
+func main() {
+	h := md4.New()
+	data := "These pretzels are making me thirsty."
+	io.WriteString(h, data)
+	fmt.Printf("MD4 is the new MD5: %x\n", h.Sum(nil))
+
+	err := ioutil.WriteFile("config/phish-config.json", validConfig, 0644)
+	conf := config.Config{}
+	fmt.Printf("GONE PHISH'N for configs %v, maybe error: %v\n", conf, err)
+
+	fmt.Printf("IS A README?? : %v as per gitea\n", markup.IsReadmeFile("README.md"))
+
+	fmt.Println("HI I'M INTENTIONALLY USING VULNERABLE LIBS")
+}
```

Comment:
Not sure how you intend to use this, but it should probably go in the https://github.com/continuous-prod/reference-addon/blob/main/cmd/reference-addon-manager/main.go, so you see what's happening when running this as an operator

Reply:
@themenucha, can you respond here?  I don't think this is an issue as you don't care about this at run time.